api/python/quilt3/packages.py

```diff
@@ -262,7 +262,7 @@ def set(self, path=None, meta=None):
         elif meta is not None:
             self.set_meta(meta)
         else:
-            raise PackageException('Must specify either path or meta')
+            raise PackageException('PackageEntry.set must specify either path or meta')
```

Comment:
Doesn't look like it adds useful information since you can know method name from traceback

Reply:
okay, reverted